pages/apim/3.x/kubernetes/apim-kubernetes-operator-overview.adoc

```diff
@@ -0,0 +1,30 @@
+[[apim-kubernetes-operator-overview]]
+= Gravitee Kubernetes Operator (GKO)
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_overview.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+
+== Overview
+
+APIM 3.19.0 has introduced the Gravitee Kubernetes Operator (GKO) - a new technical component designed to be deployed on a Kubernetes Cluster. It can also be deployed on a local cluster for testing purposes.
+
+You can use the GKO in the following ways:
+
+  * To define, deploy, and publish APIs to your API Portal and API Gateway and to manage Custom Resource Definitions (CRDs) as part of the process.
```

Comment:
> To define, deploy, and publish APIs to your API Portal and API Gateway and to manage Custom Resource Definitions (CRDs) as part of the process.

not sure of myself. 😁 
I would see something more like this :

> To define, deploy and publish APIs on your portal and API gateway and your API console with custom resource definitions (CRDs)

Reply:
Done:)